core/runtime/setup.rs

```diff
@@ -23,13 +23,11 @@ fn v8_init(
 
   let base_flags = concat!(
     " --wasm-test-streaming",
-    " --harmony-import-assertions",
-    " --harmony-import-attributes",
     " --no-validate-asm",
     " --turbo_fast_api_calls",
-    " --harmony-array-from_async",
-    " --harmony-iterator-helpers",
     " --harmony-temporal",
+    " --js-explicit-resource-management",
```

Comment:
Oh nice, last time I checked (last week) it still wasn't supported. Is it just partial support?

Reply:
Maybe. It appears that test262 is still insufficient and I'm not sure how widely it has been covered.

Reply:
V8 12.4 can parse `using` declarations but does not actually dispose the resources. The functionality was first added in V8 12.5.79.

`SuppressedError()` crashes the process.

Reply:
Thanks for the info! I've removed this flag.